scripts/aws/cfn/microservice.yml

```diff
@@ -1102,3 +1111,89 @@ Resources:
       Action: lambda:InvokeFunction
       Principal: apigateway.amazonaws.com
       SourceArn: !Sub 'arn:aws:execute-api:${AWS::Region}:${AWS::AccountId}:*/*'
+
+
+  #########################################################
+  ###        VersioningV1V21SendNewNotification         ###
+  #########################################################
+  VersioningV1V21SendNewNotificationSecGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: !Sub '${VersioningV1V21SendNewNotificationLambdaName}-sec-group'
```

Comment:
VersioningV1V21SendNewNotificationLambdaName va dichiarato in storage.yml. Questo è un esempio da un'altra lambda:

  VersioningV1V2GetNotificationLambdaName:
    Value: !Sub "${ProjectName}-delivery-versioning-v1v2-getnotification-lambda"

Reply:
in storage.yml va anche definito un log group associato alla lambda. Questo è un esempio:
  VersioningV1V2GetNotificationLambdaLogGroup:
    Type: AWS::CloudFormation::Stack
    UpdateReplacePolicy: Retain
    DeletionPolicy: Retain
    Properties:
      TemplateURL: !Sub "${TemplateBucketBaseUrl}/fragments/log-group.yaml"
      Parameters:
        LogGroupName: !Sub "${ProjectName}-delivery-versioning-v1v2-getnotification-lambda"
        LogGroupRetention: !Ref LogRetention
        LogGroupPrefix: '/aws/lambda'